projects/can_api/utils.py

```diff
@@ -1,18 +1,24 @@
+from multiprocessing.sharedctypes import Value
```

Comment:
Is this import here used for anything? If this is for the ValueError, that's a builtin, so no need to import it. This is something from the [multiprocessing](https://docs.python.org/3/library/multiprocessing.html#shared-ctypes-objects) library which doesn't look like its being used in this project

Reply:
Ah my bad, must have accidentally imported it when trying to tab complete.